it/mail/chrome/messenger/preferences/downloadactions.dtd

```diff
@@ -1,21 +1,21 @@
-<!ENTITY  downloadactionsWindow.title         "Download Actions">
+<!ENTITY  downloadactionsWindow.title         "Scarica azioni">
 <!ENTITY  window.width                         "480">
 <!ENTITY  window.height                        "310">
 
-<!ENTITY  fileTypesDescription.label          "Automatically perform the associated Action with each of the following file types:">
+<!ENTITY  fileTypesDescription.label          "Esegui automaticamente l'Azione associata con ognuno dei tipi di file seguenti:">
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+<!ENTITY  fileTypesDescription.label          "Esegui automaticamente l'Azione associata ad ognuno dei seguenti tipi di file:">